src/environments/environment.ts

```diff
@@ -26,7 +29,12 @@ interface IEnvironment {
   cbpProviderRoles: string[]
   userBucket?: string
   departments?: string[]
-  spvPath?: string
+  spvPath?: string,
```

Comment:
These keys are not added in other env file, are these already added?

Reply:
yes in other we don't need to add..we dont have interface in other env file